lib/ipam/ipam.go

```diff
@@ -1524,3 +1526,66 @@ func decideHostname(host string) (string, error) {
 	log.Debugf("Using hostname=%s", hostname)
 	return hostname, nil
 }
+
+// GetUtilization returns IP utilization info for the specified pools, or for all pools.
+func (c ipamClient) GetUtilization(ctx context.Context, args GetUtilizationArgs) ([]*PoolUtilization, error) {
+	var pools []*PoolUtilization
```

Comment:
I am a bit confused by the name `pools` here, which means PoolUtilization instead of ipPools. A better name is recommended.